pkg/api/handlers/utils/errors.go

```diff
@@ -26,7 +26,7 @@ func Error(w http.ResponseWriter, apiMessage string, code int, err error) {
 	log.Infof("Request Failed(%s): %s", http.StatusText(code), err.Error())
 	em := entities.ErrorModel{
 		Because:      (errors.Cause(err)).Error(),
-		Message:      err.Error(),
+		Message:      apiMessage,
```

Comment:
This change is going to break a lot of expectations of callers.

Reply:
If the current behavior is the way you like it, I could also just remove the apiMessage parameter and update the callers.

Reply:
I just took it out as it was just an code doc issue/unused parameter which made me wonder.
Now it is just about the nil dereferencing at the network endpoint.